pkg/crc/preset/preset.go

```diff
@@ -6,35 +6,23 @@ import (
 	"github.com/code-ready/crc/pkg/crc/logging"
 )
 
-type Preset int
+type Preset string
 
 const (
-	Podman Preset = iota
-	OpenShift
+	Podman    Preset = "podman"
+	OpenShift Preset = "openshift"
 )
 
-func (preset Preset) String() string {
-	switch preset {
-	case Podman:
-		return "podman"
-	case OpenShift:
-		return "openshift"
-	}
-
-	return "invalid"
-}
-
 func ParsePresetE(input string) (Preset, error) {
 	switch input {
-	case Podman.String():
+	case string(Podman):
 		return Podman, nil
-	case OpenShift.String():
+	case string(OpenShift):
 		return OpenShift, nil
```

Comment:
I'd squash `Preset: Implement stringer interface for preset type` in this commit instead of partially reverting this revert with a separate commit ^^
The main reason for reverting is that we don't want to get `preset: 1` when interacting with the config in the HTTP API, but `preset: "podman"`